README.md

```diff
@@ -1,4 +1,109 @@
-# lto-api.java
- LegalThings One client for Java
+LTO Network client for PHP
+===
 
-**WIP - Please [comment on this issue](https://github.com/ltonetwork/lto-api.java/issues/5) if you'd like to connect to LTO Network from Java.**
+_Signing and addresses work for the public chain only._
+
+Requirements
+---
+
+- JDK 11 or later
+
+Accounts
+---
+
+### Creation
+
+#### Create an account from seed
+
+```java com.ltonetwork.client.core
+String seed = "my seed phrase";
+
+AccountFactory af = new AccountFactory("Testnet");
+Account acc = af.seed(seed);
+```
+
+#### Create an account from sign and encrypt public keys
+
+```java com.ltonetwork.client.core
+Key signKey = new Key("mySignKeyValue", Encoding.BASE58);
+Key encryptKey = new Key("myEncryptKeyValue", Encoding.BASE58);
+
+AccountFactory af = new AccountFactory("Testnet");
+Account acc = af.createPublic(signKey, (byte) 84, encryptKey);
+```
+
+#### Create an account from full info
+
+```java com.ltonetwork.client.core
+KeyPair signKeyPair = new KeyPair (
+    new Key("mySignPublicKey", Encoding.BASE58);
+    new Key("mySignPrivateKey", Encoding.BASE58);
+);
+KeyPair encryptKeyPair = new KeyPair (
+    Key encryptPublicKey = new Key("myEncryptPublicKey", Encoding.BASE58);
+    Key encryptPrivateKey = new Key("myEncryptPrivateKey", Encoding.BASE58);
+);
+
+AccountFactory af = new AccountFactory("Testnet");
+Account acc = af.create(
```

Comment:
The `create` method should be heavily overloaded. You can generate everything from just the sing private key. If you pass more info, it must be verified. Check the PHP lib.